### DISCUSSION TITLE
college or intern

### DISCUSSION THREAD

**Discussion by Gaurav1745** [community/community#52791] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

should i skip college for some day and join intern

**Comment by Saurabhparshar** [comment]
go  for intern

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @Gaurav1745 & @Saurabhparshar, 

GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
